tensorflow/tools/ci_build/Dockerfile.gpu

```diff
@@ -1,7 +1,12 @@
-FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
+FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu14.04
```

Comment:
OK, we discussed with @jhseu offline.
It looks like somehow the merge back does not capture all the changes that need to be done to get us back to ubuntu 14.04
Once we do the merge back, he will roll back the change to upgrade our dockerfiles to ubuntu 16.

So let's leave out this file for now as @jhseu suggested.